### PR TITLE
changed link title

### PR DESCRIPTION
**Pull request by sshanks-kx**
changed so people realise links are to open source projects/etc

### CODE REVIEW

File: docs/interfaces/ffi.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ The main purpose of the library is to build stable interfaces on top of external
 
 No compiler toolchain or writing C/C++ code is required to use this library.
 
-The FFI interface is is documented and available for download [here](https://github.com/KxSystems/ffi/)
+The FFI interface is is documented and available for download from [https://github.com/KxSystems/ffi/](https://github.com/KxSystems/ffi/)
```
Comment (natalietanner) [inline docs/interfaces/ffi.md]:
```suggestion
The FFI interface is documented and available to download from [https://github.com/KxSystems/ffi/](https://github.com/KxSystems/ffi/)
```

File: docs/interfaces/csharp.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ keywords: csharp, windows, api, ipc
 
 # C# client for kdb+
 
-A kdb+ interface for the C# programming language is documented and available for download [here](https://github.com/KxSystems/csharpkdb).
+A kdb+ interface for the C# programming language is documented and available for download from [https://github.com/KxSystems/csharpkdb](https://github.com/KxSystems/csharpkdb).
```
Comment (natalietanner) [inline docs/interfaces/csharp.md]:
```suggestion
A kdb+ interface for the C# programming language is documented and available to download from [https://github.com/KxSystems/csharpkdb](https://github.com/KxSystems/csharpkdb).
```

File: docs/interfaces/java.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ keywords: java, api, ipc
 
 # Java client for kdb+
 
-A kdb+ interface for the Java programming language is documented and available for download [here](https://github.com/KxSystems/javakdb).
+A kdb+ interface for the Java programming language is documented and available for download from [https://github.com/KxSystems/javakdb](https://github.com/KxSystems/javakdb).
```
Comment (natalietanner) [inline docs/interfaces/java.md]:
```suggestion
A kdb+ interface for the Java programming language is documented and available to download from [https://github.com/KxSystems/javakdb](https://github.com/KxSystems/javakdb).
```

File: docs/interfaces/python.md [hunk 4]
```diff
@@ -14,5 +14,5 @@ PyKX supports three principal use cases:
 2. It allows users to query external q processes via an IPC interface.
 3. It allows users to embed Python functionality within a native q session using it's under q functionality.
 
-It is documented and available for download [here](https://code.kx.com/pykx).
+It is documented and available for download from [https://code.kx.com/pykx](https://code.kx.com/pykx).
```
Comment (natalietanner) [inline docs/interfaces/python.md]:
```suggestion
It is documented and available to download from [https://code.kx.com/pykx](https://code.kx.com/pykx).
```

File: docs/interfaces/rust.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ keywords: rust, api, ipc
 
 # Rust client for kdb+
 
-A kdb+ interface for the Rust programming language is documented and available for download [here](https://github.com/KxSystems/kxkdb).
+A kdb+ interface for the Rust programming language is documented and available for download from [https://github.com/KxSystems/kxkdb](https://github.com/KxSystems/kxkdb).
```
Comment (natalietanner) [inline docs/interfaces/rust.md]:
```suggestion
A kdb+ interface for the Rust programming language is documented and available to download from [https://github.com/KxSystems/kxkdb](https://github.com/KxSystems/kxkdb).
```

File: docs/kb/timezones.md [hunk 6]
```diff
@@ -57,8 +57,8 @@ q)show ttz[enlist `$"America/New_York";enlist `$"Europe/Zurich";enlist .z.P]
 
 ### Via TimeZoneDB
 
-TimeZoneDB provides a CSV generated from [IANA tz database](https://data.iana.org/time-zones/tz-link.html) which can be downloaded from [here](https://timezonedb.com/download). 
-Please check any current license details [here](https://timezonedb.com);
+TimeZoneDB provides a CSV generated from [IANA tz database](https://data.iana.org/time-zones/tz-link.html) which can be downloaded from [https://timezonedb.com/download](https://timezonedb.com/download).
```
Comment (natalietanner) [inline docs/kb/timezones.md]:
```suggestion
TimeZoneDB provides a `.csv` file generated from [IANA tz database](https://data.iana.org/time-zones/tz-link.html), which can be downloaded from [https://timezonedb.com/download](https://timezonedb.com/download). 
```

**Review by natalietanner — CHANGES_REQUESTED** [review]
Style guide fixes.

**Comment by sshanks-kx** [comment]
changes made, thanks

**Comment by natalietanner** [comment]
Approved.

**Pull request merged**